src/pivot-table/hooks/use-column-width.ts

```diff
@@ -110,11 +118,11 @@ export default function useColumnWidth(
           }),
         );
       } else {
-        const { qFallbackTitle, qApprMaxGlyphCount, columnWidth } = qDimensionInfo;
+        const { title, qApprMaxGlyphCount, columnWidth } = cell;
         const iconWidth = !isFullyExpanded && index < qNoOfLeftDims - 1 ? EXPAND_ICON_WIDTH : 0;
         const fitToContentWidth = Math.max(
-          measureTextForHeader(qFallbackTitle) + TOTAL_CELL_PADDING,
-          estimateWidthForDimensionValue(qApprMaxGlyphCount) + iconWidth,
+          measureTextForHeader(title) + TOTAL_CELL_PADDING,
+          estimateWidthForDimensionValue(qApprMaxGlyphCount as number) + iconWidth,
```

Comment:
is there a need to case `as number` here? 
since the type is number and seems like we should always have it's value according [to here](https://github.com/qlik-oss/sn-pivot-table/blob/1a37c577fdcded0081f76ce08de681726bd91d12/src/pivot-table/data/extract-headers.ts#L42) 

I noticed in types [that is optional](https://github.com/qlik-oss/sn-pivot-table/blob/1a37c577fdcded0081f76ce08de681726bd91d12/src/types/types.ts#L28) but is it because of that you need to cast it here?

Reply:
Yes. I'm casting it becaus it is optional. I know that it is defined when the cell isn't a pseudo dimension which I already checked above